package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jstransformer-foo",
-  "version": "0.0.0",
+  "private": true,
```

Comment:
Good call... We'll have to remember to remove private from the packages that use this. :+1: